frontend/components/instructionsComponent/navigation/footer/index.tsx

```diff
@@ -1,23 +1,77 @@
-import styles from "./footer.module.css";
+import React from "react";
+import { Box, Flex, Text, Link, IconButton, Select } from "@chakra-ui/react";
+import { ExternalLinkIcon } from "@chakra-ui/icons";
+import { inter } from "@/app/fonts";
+import { FaSquareGithub } from "react-icons/fa6";
+import { getNetworkConfig } from "@/constants/networks";
 
-export default function Footer() {
+const Footer: React.FC = () => {
```

Comment:
Minor, but is React.FC the latest way to do things?

Reply:
Not sure if latest, I think React.FC includes Children implicitly other that It is the same?